<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="IITL0005" failures="1" tests="1" name="test_suites.TS_UI_PMTOOL_04.TC_UI_PMTOOL_04_004" time="18.007" errors="0" timestamp="2021-10-22T12:24:19 EEST" skipped="0">
  <testcase classname="test_suites.TS_UI_PMTOOL_04.TC_UI_PMTOOL_04_004" name="TC_UI_PMTOOL_04_004_Move_Tasks" time="18.007">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at org.testng.Assert.assertFalse(Assert.java:77)
at pages.dashboard.ViewTasks.verifyMoveTask(ViewTasks.java:55)
at test_suites.TS_UI_PMTOOL_04.TC_UI_PMTOOL_04_004.TC_UI_PMTOOL_04_004_Move_Tasks(TC_UI_PMTOOL_04_004.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at Main.runOneTestSuite(Main.java:55)
at Main.runTestSuite(Main.java:36)
at Main.main(Main.java:19)
]]>
    </failure>
  </testcase> <!-- TC_UI_PMTOOL_04_004_Move_Tasks -->
  <system-out>
1. Open Web Application. PMTool home page opens 2. Navigate to Login page and fill the form 3. Verify that the login has been completed 4. Create new project 5. Verify that the project has been created 6. Create new task 7. Verify that the task has been created 8. Navigate to dashboard page and create a new task 10. Move the task to a different status. 11. Verify that the task has move  </system-out>
</testsuite> <!-- test_suites.TS_UI_PMTOOL_04.TC_UI_PMTOOL_04_004 -->
